include/circt/Dialect/Verif/VerifOps.td

```diff
@@ -260,4 +263,85 @@ def YieldOp : VerifOp<"yield", [
   ];
 }
 
+//===----------------------------------------------------------------------===//
+// Verification intent ops
+//===----------------------------------------------------------------------===//
+
+
+def FormalOp : VerifOp<"formal", [
+  NoTerminator, RegionKindInterface, HasParent<"mlir::ModuleOp">, 
+  IsolatedFromAbove, Symbol
+]> {
+  let summary = "defines a formal verification test";
+  let description = [{
+    This operation defines a formal verification test. This should be written 
+    by declaring symbolic values that are then connected to a hardware instance. 
+    These symbols can then be used in additional assertions that are defined outside 
+    of the instanciated `hw.module` but inside of this region. Assertions/Assumptions 
+    defined within the instanciated module will also be used for verification. 
+    The region will then be converted into btor2, SystemVerilog, or used for verification 
+    directly in circt-bmc. The operations in this region are ignored during regular SystemVerilog 
+    emission.   
```

Comment:
* 80 cols
* Can we briefly say why this is ignored in regular SystemVerilog? Is it because it cannot be expressed in SV or for another reason?